### DISCUSSION TITLE
Cara menambahkan kolom baru di R studio

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#533] [Q&A] [answered]
Saya punya data seperti ini :
<img width="514" alt="image" src="https://user-images.githubusercontent.com/106010284/216918910-8475f309-cab7-463b-8c02-a96b1657a96c.png">

Bagaimana cara membuat kolom baru yang isinya frekuensi dari variabel "name" ? 
Outputnya mau seperti ini
<img width="682" alt="image" src="https://user-images.githubusercontent.com/106010284/216919388-80b0a1b7-25ed-4cd0-82a1-d9c948ca3a66.png">

Terima kasih

**Comment by kevbow** [comment]
Dear @yohanespm77,

Terima kasih atas pertanyaan yang disampaikan, pertanyaan Anda sudah kami terima. 
Kami akan berusaha mencarikan referensi yang berkaitan dengan pertanyaan ini.

**Comment by kevbow** [comment, marked as answer]
Dear @yohanespm77,

Anda dapat menggunakan dplyr untuk hal ini. Berikut ini saya lampirkan kode serta screenshot hasilnya:
```r
data <- data %>% 
  group_by(name) %>% 
  mutate(freq = n()) %>% 
  data.frame()
```

![image](https://user-images.githubusercontent.com/68848168/217144244-55091f7f-79f9-4a19-890e-3249a5ae5bc4.png)

Demikian yang dapat saya sampaikan, semoga membantu.

**Reply by yohanespm77** [reply]
Terima kasih atas jawabannya Mas Kevin. Sangat membantu 👍
